queries/starlark/highlights.scm

```diff
@@ -256,6 +255,11 @@
 
 ;; Starlark-specific
 
+;; Assertion calls
+(assert_keyword) @keyword
+
+(assert_builtin) @function.builtin
```

Comment:
What's the difference between these two?
Can you use `@exception` for either or both?

Reply:
```
assert a = b
assert_eq(a = b)
assert.eq(a = b)
```

Are assert functions normally captured under `@exception`?

Reply:
Probably not the functions, but the first `assert` should be `@exception`.

Reply:
Python has it as `@keyword`

Reply:
Python has it as `@keyword`

Reply:
We might be using it inconsistently.
Let's keep it consistent with python here.

Reply:
Agreed, especially since Starlark is just a dialect of Python